### DISCUSSION TITLE
Only use DAITA on unmetered connections

### DISCUSSION THREAD

**Discussion by SkewedZeppelin** [mullvad/mullvadvpn-app#8931] [Feature requests & Ideas]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

DAITA inherently increases data used by design.
It would be nice to have an option to only use when on unmetered connection

### Alternative solutions

The alternative is not using it entirely, but that is ungood.

### Type of feature

- [ ] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [ ] Easier to use
- [X] Other

### Operating System

- [X] Android
- [X] iOS
- [ ] Windows
- [ ] macOS
- [x] Linux

**Comment by Pururun** [comment]
Thank you for your suggestion, I have created an internal issue for it.

**Comment by Rawa** [comment]
I've changed this issue to only focus on the unmetered connection characteristics to not have 2 feature requests in a single discussion. If you want a more lightweight feel free to open another discussion. As of right now the research that has been made lead to the following implementation, any changes to how DAITA works would require new research.

**Comment by UntitledDistribution** [comment]
I was about to submit another feature request for it. but apparently this request already exists. I'll paste what i was about to submit under this comment.

Feature Description:

Hi, I have recently came across a bit of an issue with my own 4G modem, its not that i do not have enough data allowance, but rather when DAITA option is enabled, it causes my device to overheat, as a solution i though of is having a toggle in the DAITA page to create an exception when you are on a metered wifi network (when setting the network as metered in your device's settings) so DAITA stays disabled when connected to metered connections but when im connected to a network that either isnt specified / is specified as Unmetered, to keep DAITA enabled during connection session. I found that it could help with slow public wifi connections or to reduce network level throttling by simply not using as much bandwidth as your device would when using DAITA.

Thank you, have a good day.

Alternative Solution:
If its not possible for android applications to fetch info such as the saved networks list but you are able to view currently connected SSID, maybe add an option so when we go into a new menu in Settings --> DAITA --> "Metered networks" --> then display the current SSID we are connected to and add a down arrow or some sort of symbol to indicate EXCEPTION to DAITA for when DAITA is enabled, this can help save internet on metered connections such as when connected to a low powered 4g wifi modem (the issue i've noticed is that it overheats and stops to work in hot temperatures which is why i'm adding feature request - and it works when DAITA is off, but i still want DAITA to be enabled for all other networks.)

Type:
- Privacy / Anonymity as it allows DAITA to still be enabled for Unmetered networks and prevent any potential threat compromising privacy, i.e. Using specialised tools to see what exactly i'm doing.
 - Easier to use
 - Other (Prevents possible network-level throttling)
 
 Operating System: Universal (you can connect any device to a public network so i say all devices Mullvad is available on)

Thank you for spending your time reading what seems to be a whole lecture from me.

**Reply by Pururun** [reply]
Thanks for your feature suggestion! It is definitely something we have discussed, we just need to figure out a way to make it work.
